module/src/link.rs

```diff
@@ -65,28 +65,49 @@ pub trait Linkable: Send + Sync {
 /// [`Linkable`]: ./trait.Linkable.html
 /// [`send`]: ./trait.Port.html#tymnethod.send
 /// [`receive`]: ./trait.Port.html#tymnethod.receive
-pub trait Port: Any + 'static {
-    /// Sets to send a list of handles to the other end on link.
-    ///
-    /// `desc` is encoded in `CBOR`.
-    fn send(&mut self, desc: &[u8]) -> &mut dyn Port;
+pub trait Port: CastFrom {
+    /// Sets to send a list of handles represented by the `ids` to the other end on link
+    /// creation. The `ids` are indices into a list of service objects created when the module
+    /// owning this port is loaded into a sandbox.CBOR map fed
+    /// to the constructor function.
+    fn export(&mut self, ids: &[usize]) -> &mut dyn Port;
```

Comment:
In my implementation of FML, a handle is represented with more info than just `usize`. May I update this later?

Reply:
`usize` is just an index into an array of service objects crated at module creation time. It's not specific to a link implementation.